assets/javascripts/discourse/components/ai-indicator-wave.gjs

```diff
@@ -0,0 +1,13 @@
+import { array } from "@ember/helper";
+
+const AiIndicatorWave = <template>
+  {{#if @loading}}
+    <span class="ai-indicator-wave">
+      {{#each (array "." "." ".") as |dot|}}
```

Comment:
🏌️ but we could just define the array at the top level and then drop it into `#each`. That way we don't need the `array` helper